# sonic needs libfftw3 so add to the configure check 
# and fixed missing separator in the Makefile
# SEP 3rd 2020 by B. Hajgato (UGent)
--- configure.ac.orig	2019-11-29 17:50:54.000000000 +0100
+++ configure.ac	2020-09-03 11:29:21.961205274 +0200
@@ -232,7 +232,7 @@
 else
     AC_CHECK_HEADERS([sonic.h],
         [
-            AC_CHECK_LIB([sonic], [sonicCreateStream])
+            AC_CHECK_LIB([sonic -lfftw3], [sonicCreateStream])
             have_sonic=yes
         ],[
             have_sonic=no
--- Makefile.am.orig	2020-09-03 12:02:45.023133962 +0200
+++ Makefile.am	2020-09-03 12:04:48.778209845 +0200
@@ -106,7 +106,7 @@
 src/speak-ng.1: src/speak-ng.1.ronn
 	ronn --roff $<
 
-docs_MARKDOWN != ls docs/*.md docs/*/*.md docs/*/*/*.md
+	docs_MARKDOWN != ls docs/*.md docs/*/*.md docs/*/*/*.md
 docs_HTML = ${docs_MARKDOWN:.md=.html}
 
 docs:	$(docs_HTML) \
